sdk/src/main/scala/com/stratio/sparkta/sdk/ValidatingPropertyMap.scala

```diff
@@ -37,9 +37,25 @@ class ValidatingPropertyMap[K, V](val m: Map[K, V]) {
         throw new IllegalStateException(s"$key is mandatory")
     }
 
-  def getConnectionConfs(key: K): String = {
+  def getConnectionConfs(key: K, firstJsonItem: String, secondJsonItem: String): String = {
     val conObj = getConnectionChain(key)
-    conObj.map(c => c.get("host").get + ":" + c.get("port").get).mkString(",")
+    conObj.map(c => c.get(firstJsonItem).get + ":" + c.get(secondJsonItem).get).mkString(",")
+  }
+
+  def getZkConnectionConfs(key: K, defaultHost: String, defaultPort: String): (String, String) = {
```

Comment:
This function must be in the file KafkaInput.scala, and their test moved to the input.